Tasks/KubernetesManifestV0/src/utils/utilities.ts

```diff
@@ -59,8 +59,10 @@ export function checkForErrors(execResults: IExecSyncResult[], warnIfError?: boo
     if (execResults.length !== 0) {
         let stderr = '';
         execResults.forEach(result => {
-            if (result.stderr) {
+            if (result.code !== 0 && result.stderr) {
                 stderr += result.stderr + '\n';
+            } else if (result.stderr) {
```

Comment:
Redundant check for if (result.stderr). You can change it to something like 
if (result.stderr) { 
                if (result.code !== 0) {stderr += result.stderr + '\n';} else { tl.warning(result.stderr);}